src/pages/organization/[id]/components/overview/PastRestorationExperience.tsx

```diff
@@ -89,10 +88,14 @@ const PastRestorationExperience = ({ organization }: PastRestorationExperiencePr
           condition={!!organization?.historic_monitoring_geojson}
         >
           <div>
-            <Text variant="text-body-500" className="w-1/3">
-              {t("Historic Monitoring Shapefile Upload:")}
+            <Text variant="text-body-900" className="w-1/3">
+              <>
+                Historic Monitoring Shapefile Upload:
```

Comment:
This one is not admin-only though, so please bring back the translation this change removed.